labelbox/schema/media_type.py

```diff
@@ -41,8 +43,10 @@ def is_supported(cls, value):
     @classmethod
     def get_supported_members(cls):
         return [
-            item for item in cls.__members__
-            if item not in ["Unknown", "Unsupported"]
+            item for item in cls.__members__ if item not in [
```

Comment:
Do we need to add LLM types as unsupported types here? Shouldn't adding the media types make `get_projects()` work with LLM projects?

Reply:
It will work without it, but tests started failing.

Reply:
@pd-lb do you remember which test started failing?